core/src/main/scala/org/broadinstitute/dsde/rawls/model/ErrorReport.scala

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.dsde.rawls.model
+
+object ErrorReport extends ErrorReportBase {
+  def SOURCE = "rawls"
+}
```

Comment:
@dvoet yes, that's what I was referring to. But I'm still not convinced it
needs to be.